token-metadata/program/src/processor.rs

```diff
@@ -255,7 +257,7 @@ pub fn process_create_metadata_accounts_v2<'a>(
     let payer_account_info = next_account_info(account_info_iter)?;
     let update_authority_info = next_account_info(account_info_iter)?;
     let system_account_info = next_account_info(account_info_iter)?;
-    let rent_info = next_account_info(account_info_iter)?;
+    let _rent_info = next_account_info(account_info_iter)?;
```

Comment:
The plan here was to only remove from the most recent handlers. Sorta as a "you should upgrade to the new handlers because it removes rent now" kinda thing. Like an incentive to upgrade I guess...

Reply:
Line `260` has no effect since the variable is not used – it can be safely removed as @austbot pointed out.

Reply:
We have other unused vars too that were in place before this PR. Only reason I kept this one in is to add additional incentives to upgrade to the latest version of handlers. I know it can be safely removed.

Reply:
It doesn't. Removing.